pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -436,14 +438,17 @@ func validateLiveUpdateFeatures(field *k8sfield.Path, spec *v1.VirtualMachineSpe
 		})
 	}
 
-	if spec.LiveUpdateFeatures != nil && spec.LiveUpdateFeatures.CPU != nil {
+	if spec.LiveUpdateFeatures != nil {
```

Comment:
Is this true with the affinity updates?

Reply:
Yes, LiveUpdateAffinity is not compatible with instance types.